docs/deployment.md

```diff
@@ -27,225 +28,301 @@ Configuration of the PDP is accomplished through two mechanisms:
+that is equivalent to the production deployment. The infrastructure is in 
+`docker/local-run/`.
+
+The infrastructure creates 4 docker containers:
+
+- PDP frontend (`pdp-local-run-fe`)
+- PDP backend (`pdp-local-run-be`)
+- PGbouncer (`pgbouncer-dev`): Manages connections to the databases.
+- Nginx proxy (`pdp-dev-proxy`): Proxies frontend and backend containers to
+  a single domain (`pdp.localhost:<port>`) to avoid CORS issues.
+  
+The frontend and backend containers mount your local codebase and install it,
+so that changes you make dynamically to the code are reflected inside the
+container as you work.
+
+The running app is available at http://pdp.localhost:5000/portal/<name>/map/;
+for example, http://pdp.localhost:5000/portal/bc_prism/map/.
+
+### Instructions
+
+1. **Advance prep**
+
+    Do each of the following things *once per workstation*.
+    You may already have done them previously, for this service or
+    another one.
+    
+    1. Set up user mapping in your local Docker daemon. 
+    Follow the instructions in 
+     [Running tests and PDP application on workstation - Procedure](https://github.com/pacificclimate/pdp-docker#procedure).
+       
+     This procedure
+     - establishes a user on your workstation named `dockeragent`,
+     - makes it a member of your own user group so that it has access to
+       your filesystem, and
+     - sets up your Docker daemon to map user `dockeragent` inside containers
+     onto `dockeragent` in your workstation. 
+    
+    1. Create `docker/local-run/common-with-passwords.env` from 
+    `docker/local-run/common.env` by adding passwords for the `pcic_meta` and 
+    `crmp` databases.
+    
+    1. Create `docker/local-run/pgbounce_users-with-passwords.txt` from  
+    `docker/dev-local/pgbounce_users.txt` by inserting correct md5 sums.
+    
+    1. Edit your `/etc/hosts`: Add `pdp.localhost` to the line starting with 
+    `127.0.0.1`. The result will look like 
+    
+        ```
+        127.0.0.1       localhost pdp.localhost
+        ```
+        This allows the Nginx reverse proxy set up by the docker-compose 
+        to refer to the domain `pdp.localhost`. Note that the frontend container is 
+        configured with `APP_ROOT` and `DATA_ROOT` using this domain.
+
+1. **Build the image**
+
+    The image need only be (re)built when:
+    
+    1. the project is first cloned, or
+    2. any of the `*requirements.txt` files change, or
+    3. the local-run Dockerfile changes.
+    
+    The built image contains all dependencies specified in those files (but not 
+    the PDP codebase). It forms the basis for installing and running your local 
+    codebase.
+    
+    To build the image:
+    
+    ```
+    docker-compose -f docker/local-run/docker-compose.yaml build
+    ```
+    
+    Image build can take several minutes.
+
+1. **Mount the gluster `/storage` volume**
+   
+    Mount locally to `/storage` so that those data files are accessible on 
+    your workstation.
```

Comment:
Could be helpful to include the `mount` command here. I know I always forget how to write it.